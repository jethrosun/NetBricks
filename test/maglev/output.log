Going to start with configuration Configuration: name:  =========== Running ZCSI Maglev ============   mempool size: 2047 core cache: 32 primary core: 1
 Ports:
	Port 0000:01:00.0 RXQ_Count: 1 RX_Queues: [ 2 ] TXQ_Count: 1 TX_Queues: 2 RXD: 128 TXD: 128 Loopback false
	Port 0000:01:00.1 RXQ_Count: 1 RX_Queues: [ 3 ] TXQ_Count: 1 TX_Queues: 3 RXD: 128 TXD: 128 Loopback false
Cores:
	2
	3
	4
	5

EAL: Probing VFIO support...
Running on node 0
Devname: "0000:01:00.0"
EAL: PCI device 0000:01:00.0 on NUMA socket -1
EAL:   Invalid NUMA socket, default to 0
EAL:   probe driver: 8086:1583 net_i40e
Going to try and use port 0
Devname: "0000:01:00.1"
EAL: PCI device 0000:01:00.1 on NUMA socket -1
EAL:   Invalid NUMA socket, default to 0
EAL:   probe driver: 8086:1583 net_i40e
Going to try and use port 1
Running on node 0
Running on node 0
Running on node 0
Receiving started
Generating permutations
Running on node 0
Receiving started
Running 0 pipelines
Receiving started
Running 0 pipelines
Receiving started
Generating permutations
Generating LUT
Generating LUT
Done Generating LUT
Running 1 pipelines
Done Generating LUT
Running 1 pipelines
Starting scheduler on 3
Starting scheduler on 4
Starting scheduler on 5
Starting scheduler on 2
0 OVERALL RX 0.00 TX 0.00 CYCLE_PER_DELAY 0 0 0
60.01 OVERALL RX 12588892.49 TX 12219099.36
