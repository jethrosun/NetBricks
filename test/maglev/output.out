Going to start with configuration Configuration: name:  =========== Running ZCSI Maglev ============   mempool size: 2047 core cache: 32 primary core: 0
 Ports:
	Port 0000:01:00.1 RXQ_Count: 1 RX_Queues: [ 5 ] TXQ_Count: 1 TX_Queues: 5 RXD: 128 TXD: 128 Loopback false
	Port 0000:01:00.0 RXQ_Count: 1 RX_Queues: [ 4 ] TXQ_Count: 1 TX_Queues: 4 RXD: 128 TXD: 128 Loopback false
Cores:
	4
	5

EAL: Probing VFIO support...
Running on node 0
Devname: "0000:01:00.1"
EAL: PCI device 0000:01:00.1 on NUMA socket -1
EAL:   Invalid NUMA socket, default to 0
EAL:   probe driver: 8086:1583 net_i40e
Going to try and use port 0
Devname: "0000:01:00.0"
EAL: PCI device 0000:01:00.0 on NUMA socket -1
EAL:   Invalid NUMA socket, default to 0
EAL:   probe driver: 8086:1583 net_i40e
Going to try and use port 1
Running on node 0
Receiving started
Running on node 0
Receiving started
Generating permutations
Generating permutations
Generating LUT
Generating LUT
Done Generating LUT
Done Generating LUT
Running 1 pipelines
Running 1 pipelines
Starting scheduler on 4
Starting scheduler on 5
0 OVERALL RX 0.00 TX 0.00 CYCLE_PER_DELAY 0 0 0
START insert for pkt count 1: Instant { tv_sec: 991087, tv_nsec: 930334948 }
START insert for pkt count 2: Instant { tv_sec: 991087, tv_nsec: 930337545 }
START insert for pkt count 3: Instant { tv_sec: 991087, tv_nsec: 930338453 }
START insert for pkt count 4: Instant { tv_sec: 991087, tv_nsec: 930339602 }
START insert for pkt count 5: Instant { tv_sec: 991087, tv_nsec: 930340407 }
START insert for pkt count 6: Instant { tv_sec: 991087, tv_nsec: 930341185 }
START insert for pkt count 7: Instant { tv_sec: 991087, tv_nsec: 930341974 }
START insert for pkt count 8: Instant { tv_sec: 991087, tv_nsec: 930342995 }
START insert for pkt count 9: Instant { tv_sec: 991087, tv_nsec: 930343801 }
START insert for pkt count 10: Instant { tv_sec: 991087, tv_nsec: 930344579 }
START insert for pkt count 11: Instant { tv_sec: 991087, tv_nsec: 930345381 }
START insert for pkt count 12: Instant { tv_sec: 991087, tv_nsec: 930346184 }
START insert for pkt count 13: Instant { tv_sec: 991087, tv_nsec: 930346960 }
START insert for pkt count 14: Instant { tv_sec: 991087, tv_nsec: 930347745 }
START insert for pkt count 15: Instant { tv_sec: 991087, tv_nsec: 930348566 }
START insert for pkt count 16: Instant { tv_sec: 991087, tv_nsec: 930349352 }
START insert for pkt count 17: Instant { tv_sec: 991087, tv_nsec: 930350127 }
START insert for pkt count 18: Instant { tv_sec: 991087, tv_nsec: 930350900 }
START insert for pkt count 19: Instant { tv_sec: 991087, tv_nsec: 930351681 }
START insert for pkt count 20: Instant { tv_sec: 991087, tv_nsec: 930352451 }
START insert for pkt count 21: Instant { tv_sec: 991087, tv_nsec: 930353226 }
START insert for pkt count 22: Instant { tv_sec: 991087, tv_nsec: 930354000 }
START insert for pkt count 23: Instant { tv_sec: 991087, tv_nsec: 930354773 }
START insert for pkt count 24: Instant { tv_sec: 991087, tv_nsec: 930355549 }
START insert for pkt count 25: Instant { tv_sec: 991087, tv_nsec: 930356369 }
START insert for pkt count 26: Instant { tv_sec: 991087, tv_nsec: 930357152 }
START insert for pkt count 27: Instant { tv_sec: 991087, tv_nsec: 930357923 }
START insert for pkt count 28: Instant { tv_sec: 991087, tv_nsec: 930358705 }
START insert for pkt count 29: Instant { tv_sec: 991087, tv_nsec: 930359487 }
START insert for pkt count 30: Instant { tv_sec: 991087, tv_nsec: 930360261 }
START insert for pkt count 31: Instant { tv_sec: 991087, tv_nsec: 930361048 }
START insert for pkt count 32: Instant { tv_sec: 991087, tv_nsec: 930361822 }
